cluster/manifests/z-karpenter/01-rbac.yaml

```diff
@@ -40,8 +40,11 @@ rules:
     verbs: [ "get", "list", "watch" ]
   # Write
   - apiGroups: ["karpenter.sh"]
-    resources: ["provisioners/status", "machines", "machines/status"]
-    verbs: ["create", "delete", "patch"]
+    resources: ["machines", "machines/status"]
+    verbs: ["create", "delete", "update", "patch"]
+  - apiGroups: ["karpenter.sh"]
+    resources: ["provisioners", "provisioners/status"]
+    verbs: ["update", "patch"]
```

Comment:
This change is from upstread from here: https://github.com/aws/karpenter/commit/4cecea29d04b34c9b7a8a2ec5261b71d8be4af4f